data_steward/analytics/tools/visualizations/plausibility/duplicates.py

```diff
@@ -71,63 +71,62 @@
         new_df[col] = new_col
 
     new_table_sheets[name] = new_df
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Condition Occurrence'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-ax.set_title("Condition Table Duplicates", size=14)
-# plt.savefig("condition_table_duplicates.jpg")
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Drug Exposure'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-ax.set_title("Drug Table Duplicates", size=14)
-# plt.savefig("drug_table_duplicates.jpg")
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Measurement'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-ax.set_title("Measurement Table Duplicates", size=14)
-# plt.savefig("measurement_table_duplicates.jpg")
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Observation'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-ax.set_title("Observation Table Duplicates", size=14)
-# plt.savefig("observation_table_duplicates.jpg")
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Procedure Occurrence'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-ax.set_title("Procedure Table Duplicates", size=14)
-# plt.savefig("procedure_table_duplicates.jpg")
-
-# +
-fig, ax = plt.subplots(figsize=(18, 12))
-sns.heatmap(new_table_sheets['Visit Occurrence'], annot=True, annot_kws={"size": 10},
-            fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
-            xticklabels=date_cols, cmap="YlGnBu")
-
-
-ax.set_title("Visit Table Duplicates", size=14)
-# plt.savefig("visit_table_duplicates.jpg")
 # -
 
+# fig, ax = plt.subplots(figsize=(18, 12))
+# sns.heatmap(new_table_sheets['Condition Occurrence'], annot=True, annot_kws={"size": 10},
+#             fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
+#             xticklabels=date_cols, cmap="YlGnBu")
+# ax.set_ylim(len(hpo_id_cols)-0.1, 0)
+#
+# ax.set_title("Condition Table Duplicates", size=14)
+# # plt.savefig("condition_table_duplicates.jpg")
+
+# fig, ax = plt.subplots(figsize=(18, 12))
+# sns.heatmap(new_table_sheets['Drug Exposure'], annot=True, annot_kws={"size": 10},
+#             fmt='g', linewidths=.5, ax=ax, yticklabels=hpo_id_cols,
+#             xticklabels=date_cols, cmap="YlGnBu")
```

Comment:
You could remove these if they are not needed to run the script. Version control will always have the old code if needed.